Include/internal/condvar.h

```diff
@@ -34,58 +34,13 @@
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
 
-/* options */
```

Comment:
Maybe you could change the comment before `#define Py_HAVE_CONDVAR` ?

Reply:
in fact, should we remove all the references to Vista when we drop the support of Vista ?

Reply:
I updated the comment.